### ISSUE TITLE
GK fails when service instantiation is requested

### ISSUE THREAD

**Issue by tsoenen** [sonata-nfv/son-gkeeper#536]
1. Package is pushed through GK api
2. GET on /services GK api
3. copy service_uuid from response
4. POST on /requests with the above service uuid

Leads to a 500 response. This is in  requests/logs: 

```
D, [2017-04-18 21:30:56#12] DEBUG -- : undefined method `each_with_index' for nil:NilClass
D, [2017-04-18 21:30:56#12] DEBUG -- : ["/app/routes/request.rb:93:in `block in <class:GtkSrv>'", "/usr/local/bundle/gems/sinatra-1.4.7/lib/sinatra/base.rb:1611:in `call'", "/usr/local/bundle/gems/sinatra-1.4.7/lib/sinatra/base.rb:1611:in `block in compile!'", "/usr/local/bundle/gems/sinatra-1.4.7/lib/sinatra/base.rb:975:in `[]'", "/usr/local/bundle/gems/sinatra-1.4.7/lib/sinatra/base.rb:975:in `block (3 levels) in route!'", "/usr/local/bundle/gems/sinatra-1.4.7/lib/sinatra/base.rb:994:in `route_eval'", "/usr/local/bundle/gems/sinatra-1.4.7/lib/sinatra/base.rb:975:in `block (2 levels) in route!'", "/usr/local/bundle/gems/sinatra-1.4.7/lib/sinatra/base.rb:1015:in `block in process_route'", "/usr/local/bundle/gems/sinatra-1.4.7/lib/sinatra/base.rb:1013:in `catch'", "/usr/local/bundle/gems/sinatra-1.4.7/lib/sinatra/base.rb:1013:in `process_route'", "/usr/local/bundle/gems/sinatra-1.4.7/lib/sinatra/base.rb:973:in `block in route!'", "/usr/local/bundle/gems/sinatra-1.4.7/lib/sinatra/base.rb:972:in `each'", "/usr/local/bundle/gems/sinatra-1.4.7/lib/sinatra/base.rb:972:in `route!'", "/usr/local/bundle/gems/sinatra-1.4.7/lib/sinatra/base.rb:1085:in `block in dispatch!'", "/usr/local/bundle/gems/sinatra-1.4.7/lib/sinatra/base.rb:1067:in `block in invoke'", "/usr/local/bundle/gems/sinatra-1.4.7/lib/sinatra/base.rb:1067:in `catch'", "/usr/local/bundle/gems/sinatra-1.4.7/lib/sinatra/base.rb:1067:in `invoke'", "/usr/local/bundle/gems/sinatra-1.4.7/lib/sinatra/base.rb:1082:in `dispatch!'", "/usr/local/bundle/gems/sinatra-1.4.7/lib/sinatra/base.rb:907:in `block in call!'", "/usr/local/bundle/gems/sinatra-1.4.7/lib/sinatra/base.rb:1067:in `block in invoke'", "/usr/local/bundle/gems/sinatra-1.4.7/lib/sinatra/base.rb:1067:in `catch'", "/usr/local/bundle/gems/sinatra-1.4.7/lib/sinatra/base.rb:1067:in `invoke'", "/usr/local/bundle/gems/sinatra-1.4.7/lib/sinatra/base.rb:907:in `call!'", "/usr/local/bundle/gems/sinatra-1.4.7/lib/sinatra/base.rb:895:in `call'", "/usr/local/bundle/gems/rack-1.6.5/lib/rack/session/abstract/id.rb:225:in `context'", "/usr/local/bundle/gems/rack-1.6.5/lib/rack/session/abstract/id.rb:220:in `call'", "/usr/local/bundle/gems/rack-protection-1.5.3/lib/rack/protection/xss_header.rb:18:in `call'", "/usr/local/bundle/gems/rack-protection-1.5.3/lib/rack/protection/path_traversal.rb:16:in `call'", "/usr/local/bundle/gems/rack-protection-1.5.3/lib/rack/protection/json_csrf.rb:18:in `call'", "/usr/local/bundle/gems/rack-protection-1.5.3/lib/rack/protection/base.rb:49:in `call'", "/usr/local/bundle/gems/rack-protection-1.5.3/lib/rack/protection/base.rb:49:in `call'", "/usr/local/bundle/gems/rack-protection-1.5.3/lib/rack/protection/frame_options.rb:31:in `call'", "/usr/local/bundle/gems/rack-1.6.5/lib/rack/logger.rb:15:in `call'", "/usr/local/bundle/gems/rack-1.6.5/lib/rack/commonlogger.rb:33:in `call'", "/usr/local/bundle/gems/sinatra-1.4.7/lib/sinatra/base.rb:219:in `call'", "/usr/local/bundle/gems/sinatra-1.4.7/lib/sinatra/base.rb:212:in `call'", "/usr/local/bundle/gems/rack-1.6.5/lib/rack/head.rb:13:in `call'", "/usr/local/bundle/gems/sinatra-1.4.7/lib/sinatra/base.rb:182:in `call'", "/usr/local/bundle/gems/sinatra-1.4.7/lib/sinatra/base.rb:2013:in `call'", "/usr/local/bundle/gems/sinatra-1.4.7/lib/sinatra/base.rb:1487:in `block in call'", "/usr/local/bundle/gems/sinatra-1.4.7/lib/sinatra/base.rb:1787:in `synchronize'", "/usr/local/bundle/gems/sinatra-1.4.7/lib/sinatra/base.rb:1487:in `call'", "/usr/local/bundle/gems/puma-3.6.2/lib/puma/configuration.rb:225:in `call'", "/usr/local/bundle/gems/puma-3.6.2/lib/puma/server.rb:578:in `handle_request'", "/usr/local/bundle/gems/puma-3.6.2/lib/puma/server.rb:415:in `process_client'", "/usr/local/bundle/gems/puma-3.6.2/lib/puma/server.rb:275:in `block in run'", "/usr/local/bundle/gems/puma-3.6.2/lib/puma/thread_pool.rb:116:in `call'", "/usr/local/bundle/gems/puma-3.6.2/lib/puma/thread_pool.rb:116:in `block in spawn_thread'"]
```

If you require more info, please contact me!

**Comment by jbonnet** [comment]
Same as #543, closed by #546

**Comment by jbonnet** [comment]
Closed by #556
